components/Api/ApiChanges/index.tsx

```diff
@@ -0,0 +1,51 @@
+import { FormattedMessage } from 'react-intl';
+import styles from './index.module.scss';
+import { parseApiDocsVersion } from '../../../util/parseApiDocsVersion';
+import type { ApiChange, ApiUpdate } from '../../../types';
+import type { FC } from 'react';
+
+type ChangesProps = {
+  update: ApiUpdate;
+  changes: ApiChange[];
+};
+
+const Changes: FC<ChangesProps> = ({ update, changes }) => (
```

Comment:
Can we name this component `ApiChanges`? to be consistent with the name of the folder 🤔

Reply:
ok